Base/QTGUI/Resources/Icons/ExtensionIncompatible.svg

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
-<!-- Created with Inkscape (http://www.inkscape.org/) -->
+<!-- Created with Inkscape (https://www.inkscape.org/) -->
 
 <svg
-   xmlns:dc="http://purl.org/dc/elements/1.1/"
-   xmlns:cc="http://creativecommons.org/ns#"
-   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
-   xmlns:svg="http://www.w3.org/2000/svg"
-   xmlns="http://www.w3.org/2000/svg"
-   xmlns:xlink="http://www.w3.org/1999/xlink"
+   xmlns:dc="https://purl.org/dc/elements/1.1/"
+   xmlns:cc="https://creativecommons.org/ns#"
+   xmlns:rdf="https://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="https://www.w3.org/2000/svg"
+   xmlns="https://www.w3.org/2000/svg"
+   xmlns:xlink="https://www.w3.org/1999/xlink"
    xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
```

Comment:
Wondering why this reference was not updated, I realized that current tools are not saving new files with `https://`.

For that reason, I suggest we do not update our `svg` files.

Few examples:

### Inkscape 1.1.2 (76b9e6a115, 2022-02-25)

```
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!-- Created with Inkscape (http://www.inkscape.org/) -->

<svg
   width="210mm"
   height="297mm"
   viewBox="0 0 210 297"
   version="1.1"
   id="svg5"
   inkscape:version="1.1.2 (76b9e6a115, 2022-02-25)"
   sodipodi:docname="drawing.svg"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
....
```

### Adobe Illustrator 24.2.1

Looking at files generated (in July 2020) using `Adobe Illustrator` (for example `3D Slicer_Mark.svg`), they still reference `http://`. 

```
<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 24.2.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
         viewBox="0 0 512 513.3" style="enable-background:new 0 0 512 513.3;" xml:space="preserve">
<style type="text/css">

...
```

Reply:
Ok we can remove the svg updates here. I think I hadn’t originally updated it because https://sodipodi.sourceforge.net/ wasn’t a valid link though I guess https://sourceforge.net/projects/sodipodi/ is valid.

Reply:
These are not website links, but namespaces. They must not be changed. See for example https://stackoverflow.com/questions/30707609/xml-namespace-uri-with-https